hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveMetastore.java

```diff
@@ -233,8 +253,8 @@ private void setupMetastoreDB(String dbURL) throws SQLException, IOException {
     }
   }
 
-  private String getDerbyPath() {
-    File metastoreDB = new File(hiveLocalDir, "metastore_db");
+  private static String getDerbyPath() {
```

Comment:
nit: If the `HIVE_LOCAL_DIR` is static, we might get nicer code if we create a static `DERBY_PATH` as well.